tests/model_fields/test_onetoonefield.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
```

Comment:
Not sure about the test location. `model_inheritance` seems like a more natural place to me given the changes don't touch anything on `OneToOneField` itself.